root/src/main/java/com/example/servingwebcontent/ConnectDatabase.java

```diff
@@ -0,0 +1,69 @@
+package com.example.servingwebcontent;
+
+import java.sql.*;
+import java.util.*;
+
+public class ConnectDatabase {
+	Connection con = null;
+
+	public ConnectDatabase (String database, String user, String pass) {
+		try {
+			con = DriverManager.getConnection("jdbc:mysql://localhost/" + database + "?" + "user=" + user + "&password=" + pass);
+		} 
+		catch (SQLException e) { 
+			System.out.println(e);
+			throw new RuntimeException(); 
```

Comment:
Add a description for each of the RuntimeExceptions so it is easy to know what went wrong. Also, pass in the SQLException to RuntimeException as the Throwable parameter so that the stack trace will include the SQLException in the "caused by" section. Do that for all the other places too